packages/nuxt/src/pages/module.ts

```diff
@@ -3,8 +3,10 @@ import { addComponent, addPlugin, addTemplate, addVitePlugin, addWebpackPlugin,
 import { join, relative, resolve } from 'pathe'
 import { genImport, genObjectFromRawEntries, genString } from 'knitwork'
 import escapeRE from 'escape-string-regexp'
-import { joinURL } from 'ufo'
+import { joinURL, withoutLeadingSlash } from 'ufo'
 import type { NuxtApp, NuxtPage } from 'nuxt/schema'
+import micromatch from 'micromatch'
```

Comment:
what do you think about using the nitro route matcher, for 1:1 parity?

https://github.com/unjs/nitro/blob/8e06f2e4a65c6de0d2f2f3a5ef156ae3821ed60a/src/prerender.ts#L70-L75

Reply:
sure 🙂 :+1:

Reply:
like this ? 🤔